Fix for gcc14.  
However, on linux the removed line is needed to build the tests
so they have been excluded.

--- meson.build.orig	2024-09-14 18:26:17 UTC
+++ meson.build
@@ -8,7 +8,6 @@ project('at-spi2-core', 'c',
         ],
         meson_version: '>= 0.63.0')
 
-add_project_arguments([ '-D_POSIX_C_SOURCE=200809L', '-D_DEFAULT_SOURCE' ], language: 'c')
 
 fs = import('fs')
 
@@ -207,7 +206,6 @@ if not get_option('atk_only')
   subdir('registryd')
   subdir('droute')
   subdir('atk-adaptor')
-  subdir('tests')
 endif
 
 if get_option('docs')
